pkgs/development/python-modules/rst2pdf/default.nix

```
@@ -0,0 +1,63 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, docutils
+, importlib-metadata
+, jinja2
+, packaging
+, pygments
+, pyyaml
+, reportlab
+, smartypants
+, pillow
+, pytestCheckHook
+, pymupdf
+, sphinx
+}:
+
+buildPythonPackage rec {
+  pname = "rst2pdf";
+  version = "0.99";
+
```

```suggestion
  format = "pyproject";
```

Format is going to become a mandatory field some time after 22.11 and using `pyproject` is preferred.

It is possible you may need to configure `nativeBuildInputs`, see https://github.com/rst2pdf/rst2pdf/blob/0.99/pyproject.toml#L1-L2